src/components/performance/search-modal.tsx

```
@@ -108,37 +136,49 @@ export function SearchModal({ onSelect, trigger }: LocationSearchModalProps) {
+                                  className={`
+                                    cursor-pointer space-y-2.5 py-5
+                                    transition-colors
+                                    hover:bg-orange-50
+                                  `}
+                                  onClick={() => {
+                                    onSelect(loc);
+                                    setIsOpen(false);
+                                  }}
+                                >
+                                  <p className="typo-caption-sb-1 text-gray-800">{loc.name}</p>
+                                  <p className="typo-caption-m-1 text-gray-550">{loc.address}</p>
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)